ysql_grammar.ebnf

```diff
@@ -0,0 +1,1671 @@
+(* 'ABORT' *)
```

Comment:
why is this file checked in?

Reply:
for testing . Hoping to add more test cases with this file

Reply:
1. it should be smaller
2. It should be in a subdirectory